### PR TITLE
Feature/sample plugins: VideoInfoPlugin

### PR DESCRIPTION
**Pull request by pmunizferreira**
Goal
----
Add external plugin to Player.

- `VideoInfoPlugin` is a `MediaControl.Plugin` to show title and subtitle if informed by Options

How to Test
----
- open clappr app
- for both fullscreen and embedded modes, every time media control is clicked, title should be shown.
- only on fullscreen mode, every time media control is clicked, subtitle should be shown.

### CODE REVIEW

File: app/src/test/kotlin/io/clappr/player/app/plugin/NextVideoPluginTest.kt [hunk 1]
```diff
@@ -108,27 +115,4 @@ class NextVideoPluginTest {
 
         assertEquals(UIPlugin.Visibility.HIDDEN, nextVideoPlugin.visibility)
     }
-
-    @Test
```
Comment (UelitonFreitas) [inline app/src/test/kotlin/io/clappr/player/app/plugin/NextVideoPluginTest.kt]: Why was need to remove this test?

File: app/src/main/kotlin/io/clappr/player/app/PlayerActivity.kt [hunk 2]
```diff
@@ -21,15 +22,21 @@ class  PlayerActivity : Activity() {
...
+    private val videoTitle by lazy { findViewById<EditText>(R.id.video_title) }
+    private val videoSubtitle by lazy { findViewById<EditText>(R.id.video_subtitle) }
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_player)
 
         videoUrl.setText("http://clappr.io/highline.mp4", TextView.BufferType.EDITABLE)
+        videoTitle.setText("Highline", TextView.BufferType.EDITABLE)
+        videoSubtitle.setText("Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.", TextView.BufferType.EDITABLE)
```
[the remark this answers is not in the document]
Comment (pmunizferreira) [reply app/src/main/kotlin/io/clappr/player/app/PlayerActivity.kt]: done

File: app/src/main/kotlin/io/clappr/player/app/PlayerActivity.kt [hunk 3]
```diff
@@ -21,15 +22,21 @@ class  PlayerActivity : Activity() {
...
     private val videoUrl by lazy { findViewById<EditText>(R.id.video_url) }
+    private val videoTitle by lazy { findViewById<EditText>(R.id.video_title) }
+    private val videoSubtitle by lazy { findViewById<EditText>(R.id.video_subtitle) }
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_player)
 
         videoUrl.setText("http://clappr.io/highline.mp4", TextView.BufferType.EDITABLE)
+        videoTitle.setText("Highline", TextView.BufferType.EDITABLE)
```
[the remark this answers is not in the document]
Comment (pmunizferreira) [reply app/src/main/kotlin/io/clappr/player/app/PlayerActivity.kt]: done

File: app/src/test/kotlin/io/clappr/player/app/plugin/VideoInfoPluginTest.kt [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+    private lateinit var core: Core
+
+    private lateinit var videoInfoPlugin: VideoInfoPlugin
+
+    private val source = "url"
+    private val title = "title"
+    private val subtitle = "subtitle"
+
+    @Before
+    fun setup() {
```
[the remark this answers is not in the document]
Comment (pmunizferreira) [reply app/src/test/kotlin/io/clappr/player/app/plugin/VideoInfoPluginTest.kt]: done

File: app/src/test/kotlin/io/clappr/player/app/plugin/NextVideoPluginTest.kt (hunk 1 above)
Comment (pmunizferreira) [reply app/src/test/kotlin/io/clappr/player/app/plugin/NextVideoPluginTest.kt]: moved to top

File: app/src/main/kotlin/io/clappr/player/app/plugin/VideoInfoPlugin.kt [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+        core.activePlayback?.let {
+            playbackListenerIds.add(listenTo(it, Event.WILL_PLAY.value, Callback.wrap { show() }))
+        }
+    }
+
+    private fun updateVideoInfo() {
+        titleLabel.text = title
+        subtitleLabel.text = subtitle
+
+        if (core.fullscreenState == Core.FullscreenState.FULLSCREEN) {
```
[the remark this answers is not in the document]
Comment (pmunizferreira) [reply app/src/main/kotlin/io/clappr/player/app/plugin/VideoInfoPlugin.kt]: done

**Pull request merged**